util/ubuntu-dependencies

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This script installs streisand's dependencies on an Ubuntu 16.04
```

Comment:
s/streisand/Streisand/g

Reply:
OK. Streisand is the name of the system, and the main shell script is called `streisand` or maybe `./streisand`?